apis/cache/v1alpha3/redis_types.go

```diff
@@ -56,101 +29,138 @@ const (
 	SupportedRedisVersion = "3.2"
 )
 
-// RedisParameters define the desired state of an Azure Redis cluster. Most
-// fields map directly to an Azure Redis resource:
+// An SKU represents the performance and cost oriented properties of a
```

Comment:
```suggestion
// A SKU represents the performance and cost oriented properties of a
```

Reply:
I feel like "An SKU" is more correct here, for reasons I cannot explain. 🤔

Reply:
in my time at MSFT, it was always pronounced as a word, e.g. "skew".  i never heard it pronounced as the letters, e.g. S-K-U.

therefore, ergo, ipso facto, IMO it should be "A SKU" ("a skew")

Reply:
Googling it has shown me that the world is divided into two pieces about that 3-letter word's pronunciation. I'm choosing to opt for the one that is more appealing to the ones who have no idea about what SKU is; _AN SKU_.